INSTALL.md

```diff
@@ -132,12 +152,12 @@ cget install Dobiasd/FunctionalPlus
 ```
 
 
-### way 4: download manually
+### way 5: download manually
 
 Just [download](https://github.com/Dobiasd/FunctionalPlus/archive/master.zip)/extract FunctionalPlus and tell your compiler to use the `include` directory.
 
 
-### way 5: using [Conan C/C++ package manager](https://conan.io)
+### way 6: using [Conan C/C++ package manager](https://conan.io)
```

Comment:
Regarding Conan, this file at the moment suggests using the [`cmake`](https://docs.conan.io/en/latest/reference/generators/cmake.html) generator, however from what I have gathered, the proper generator of choice should be [`cmake_find_package_multi`](https://docs.conan.io/en/latest/reference/generators/cmake_find_package_multi.html), because that generates a real CMake package, however it is marked as experimental and it completely ignores the project's lists files, which already provide a real CMake package.

Can someone comment on this? Can Conan just use the project's CMake files without generating its own files?

Reply:
@danimtb Can you help out here? :slightly_smiling_face:

Reply:
The information here is still valid and the generator to use is up to the user's choice. No need to update anything here.

As a way to clarify about the CMake files: Conan is build-system agnostic and works with the information provided in the conanfile.py package_info(). That information is translated to the build system of choice (generators). That is the reason why packaging CMake files is not needed and even discouraged in OSS projects for better integration with build systems and to avoid some bad practices like hardcoded paths in *Config.cmake/*Find*.cmake files

Reply:
Why not drop the generator and show the line to add only?

Reply:
@friendlyanon the package reference has change as it's now available through conan center:

```
functionalplus/0.2.13-p0
```

This PR is a good change to update it :+1: .

Reply:
Updated the requires clause.

Another thing I noticed is that `conanfile.txt` is still there, but this project is instead using the `doctest` provided CMake package to be able to use `doctest_discover_tests`.

Reply:
> Updated the requires clause.

Perfect, thanks.

 `conanfile.txt` is a way to get development dependencies, eg. instead of cloning the doctest git repository.
Instead of picking an installed library, cmake can pickup the paths shipped by conan.